tests/runner.py

```diff
@@ -753,15 +766,17 @@ def get_build_dir(self):
 
   def get_library(self, name, generated_libs, configure=['sh', './configure'],
                   configure_args=[], make=['make'], make_args='help',
-                  env_init={}, cache_name_extra='', native=False):
+                  env_init={}, cache_name_extra='', native=False, cflags=[]):
     if make_args == 'help':
```

Comment:
I think all the caller of get_library pass self.get_emcc_flags().. so can you avoid adding this argument and simple have `cflags = self.get_emcc_args()` as the first line of this function?

Reply:
Thanks, simplified the cflags that way.